app/components/ui/molecules/ModalDialog.md

```diff
@@ -35,3 +40,40 @@ const FormH2 = require('../atoms/FormHeadings.js').FormH2
   </ModalDialog>
 </div>
 ```
+
+PersonInfo modal:
```

Comment:
This is failing on the styleguide, for the same reason as the `PeopleInfo` failure